src/main/scala/com/fulcrumgenomics/alignment/Alignment.scala

```diff
@@ -82,4 +120,55 @@ case class Alignment(query: Array[Byte],
 
   /** One based closed coordinate of the end of the alignment on the query sequence. */
   def targetEnd: Int = targetStart + cigar.lengthOnTarget - 1
+
+  /**
+    * Generates a padded text representation of the alignment for visualization. The returned
+    * sequence will consist of three lines as follows (minus the labels on the left):
+    *
+    * query : ACGTGAACTGACT-ACTGTATGCG
+    * align : |||||  |||||| ||||||||.|
+    * target: ACGTG--CTGACTGACTGTATGGG
```

Comment:
UCSC's BLAT output will use `.` instead of `-` for a missing base in either target/query:
```
00000051 agaaatttgggatgtacatcttccagaaattttttt..gaaatcatctga 00000098
>>>>>>>> ||||||||||||||||||||||||||||||||||||  |||||||||||| >>>>>>>>
69999771 agaaatttgggatgtacatcttccagaaatttttttttgaaatcatctga
```
I also like that it gives the start and end coordinates before and after the alignment respectively.  This would be useful for local alignments.  I am sure folks are very familiar with that text format, so perhaps it is good to try to emulate.